integration_tests/src/main/python/asserts.py

```diff
@@ -267,7 +267,7 @@ def _assert_gpu_and_cpu_writes_are_equal(
     print('### CPU RUN ###')
     cpu_start = time.time()
     cpu_path = base_path + '/CPU'
-    with_cpu_session(lambda spark : write_func(spark, cpu_path), conf=conf)
+    cpu_result = with_cpu_session(lambda spark : write_func(spark, cpu_path), conf=conf)
```

Comment:
Rename to `cpu_write_result`?  and also rename `gpu_result` to `gpu_writhe_result`?
We may need to make it clear to distinct the following `from_cpu` and `from_gpu`

Reply:
Good explanation, might help to add this comment in the method if others find it confusing.